### DISCUSSION TITLE
Add uncertainty estimates around emission factors

### DISCUSSION THREAD

**Discussion by pierresegonne** [electricitymaps/electricitymaps-contrib#4923] [Ideas]
We could increase transparency around our level of confidence for emission factors by adding uncertainty levels.

Someone has been suggesting something like this:

<img width="962" alt="Screenshot 2023-01-02 at 15 51 57" src="https://user-images.githubusercontent.com/32778266/210247374-eb5eed14-ceeb-45d7-83fb-18999393f92e.png">

I'm opening this discussion to poll whether the community shares the enthusiasm for such feature.

**Comment by VIKTORVAV99** [comment]
Maybe adding a range would be better than precentages?

For example wind would be 11-14 with a mean average of 13, then it would be hard to show precentages in a nice way.

But I'd be happy to discuss it and prototype both.

Side note: We could potentially do the same for our estimates based on historical accuracy.

**Reply by ForceTramZebra3** [reply]
Hello, I know absolutely nothing about coding, but I love your 'Electricity Map' app. It helps me enormously when in discussions with friends about green and low emission CO2 technologies, and how best various technologies should sit within our energy system here in Australia, with the ultimate aim of getting CO2 emissions down to zero. The annotation of an error range, in whatever form, either in +/-percentages or in +/- gCO2eq/kWh, would be very useful when I discuss these technologies with friends, and occasionally with politicians.  Often the question arises 'Hey, how accurate and meaningful are these figures?' To have the ability to answer such questions honestly and forthrightly, instantly raises up the discussion, and adds credibility, even if the error range is huge. 

Thankyou for your time, Regards ...

**Reply by pierresegonne** [reply]
I was originally referring to having uncertainties only for emission factors, but having those for estimates / forecasts could indeed be a good idea. That would definitiely be outside the scope of this first initiative though. 

Feel free to try to prototype around the emission factors though!

**Comment by mirkoschaefer** [comment]
Just a few thoughts (from the EU perspective, don't know much about other regions):
Generally it would increase transparency if uncertainties are given, although I fear it is hard to express them simply as ranges. Overall, your published emission intensities (both consumption and generation based) depend on a lot of factors, with the per type per country emission factors only being one part of the chain. But since these can (and are) displayed as single numbers and are a clearly identifiable input it is natural to start there.

For increasing transparency, it would make sense to discriminate between sensitivity with respect to methodological choices (like "which technologies do I group as "coal" in my data set", or "how do I account for emissions from CHP plants", or "which part of the generation fleet these factors intent to represent?") and uncertainties due to data quality when estimating emission factors given these choices.

Of course, this discussion should then be extended to the power system data (which also includes both methodological choices and data issues), because the emission factors are combined with this data for your published results. For instance, ENTSO-E per type generation data (which is part of your input) does not represent the entire electricity generation (the fraction depends on energy carrier and reporting requirements), so should your emission factors be defined according to the published generation data? Or do you determine emission factors independently from the available power system data, then emission factors and power system data do not represent the same part of the system.
